### DISCUSSION TITLE
React Components

### DISCUSSION THREAD

**Discussion by acidjunk** [fomantic/Fomantic-UI#596] [General] [answered]
# Help Wanted
I'm using Semantic 2.4 with https://react.semantic-ui.com/

## Problem
I'm not sure what to use: after some reading (and it's hidden quite well) it seems like current Semantic-UI development is almost 0: so luckily this fork exists. But before I invest time to rewrite current templates/react components I need to decide if it's worth the effort.

I really like the semantic ui react approach, but I also red info about using Fomantic UI (no more jquery and say hello to shiny new: ES6 classes).

What would you advice? What's the best way to use Formantic UI with an React project now and in the near future?

**Comment by lubber-de** [comment, marked as answer]
For now you could use Fomantic-UI together with Sui react. Here is a guide
https://medium.com/@fomantic/setting-up-fomantic-ui-with-react-f701b68f736c

**Comment by watson-test** [comment]
Thanks for the detailed info. Will look into it as I also want to theme it.

**Comment by lubber-de** [comment]
Closing because of no more feedback assuming it's solved and works

**Comment by acidjunk** [comment]
Thanks for the info. But using it this way seems to be only possible for stuff that Semantic-UI itself uses. E.g. newer, Fomantic UI, only components are not supported by Semantic-UI-React. It feels a bit to "temporary" to use it for production React apps.

**Comment by naterexw** [comment]
Here is a Next.js + Fomantic-UI + React starter template
https://github.com/skydiver/nextjs-semantic

**Comment by newearthmartin** [comment]
Hey guys I'm starting to see deprecation warnings in semantic-ui-react. I wonder if fomantic-ui plans to absorbe semantic-ui-react as well? I've been a long time user of semantic-ui and I just found out about fomantic-ui. I dont know all the details, but I understand that semantic-ui is unmaintained but development continued here in fomantic-ui. 

I opened an issue in semantic-ui-react about these warnings https://github.com/Semantic-Org/Semantic-UI-React/issues/4535
